e2etest/bgptests/infra_setup.go

```diff
@@ -125,6 +125,8 @@ func InfraSetup(ipv4Addresses, ipv6Addresses []string, cs *clientset.Clientset)
 			ASN:     externalASN,
 			BGPPort: 179,
 		},
+		HostIPv4: hostIPv4,
```

Comment:
No need to say "small fix" in the commit subject.
Just say what we are doing here (and how, in the body).

Also, same impersonal comment:

`ebgpSingleHopContainerConfig` is missing the following:
		HostIPv4: hostIPv4,
		HostIPv6: hostIPv6,

so we add them because blablabla

Reply:
Sound and clear. thanks.